src/components/header/Header.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { FunctionComponent, useState } from "react";
+import { Link } from "react-router-dom";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faBars, faTimes } from "@fortawesome/free-solid-svg-icons";
+import fsLogo from "../../assets/images/firstStepLogo.png";
+import useWindowDimensions from "../../hooks/useWindowDimensions";
+import styles from "./Header.module.scss";
+
+interface HeaderProps {}
+
+interface INav {
+  label: string;
+  url: string;
+}
+
+const Header: FunctionComponent<HeaderProps> = () => {
+  const [isOpen, useIsOpen] = useState(false);
```

Comment:
Function names starting with **use** are hooks. For e.g. `useState`, `useEffect`, `useWindowDimentions`.

Please change `useIsOpen` to `setIsOpen` or any other relevant name.